html/js/melpa.js

```diff
@@ -150,7 +151,7 @@
       pkgs.push(new melpa.Package({
         name: name,
         version: version,
-        commit: built.props.commit,
+        commit: props.commit || '',
```

Comment:
Yeah, the point is that code elsewhere should be able to say something like `if (package.props.commit) { ... }` in order to only render commit info in the page if it is present.